galasa-ui/src/components/test-runs/LogTab.tsx

```diff
@@ -383,7 +383,7 @@ export default function LogTab({ logs }: { logs: string }) {
       const filtered = applyFilters(processed);
       setProcessedLines(filtered);
     }
-  }, [logContent, filters]);
+  }, [logContent, filters, processLogLines, applyFilters]);
```

Comment:
Instead of adding more dependencies here, we could just define the processLogLines and applyFilters inside the useEffect? Assuming they won't be used outside